txpool/txpool.go

```diff
@@ -168,7 +167,7 @@ type TxPool struct {
 
 	// flag indicating if the current node is a sealer,
 	// and should therefore gossip transactions
-	sealing atomic.Bool
+	sealing uint32
```

Comment:
can't we use go 1.19 with atomic.Bool and atomic.Pointer[T]?

Reply:
I agree with @JekaMas, we can replace this with no code changes with `sync/atomic`.

Reply:
But we can't use atomic.Bool under the current version, go 1.18?

Reply:
Don't be too optimistic too early :slightly_smiling_face: 

As you can see, failing linter [here](https://github.com/0xPolygon/polygon-edge/actions/runs/3370708206/jobs/5591956495):
`Error: Bool not declared by package atomic (typecheck)`

It is related to what @Kourin1996 is suggesting: we are running go 1.18 on Edge, but `atomic.Bool` is introduced as a part of go 1.19.

Reply:
But it does compile for me locally. Ok, I have `go version go1.19.1 linux/amd64`.

Reply:
@ferranbt @Stefan-Ethernal Can we create a task to upgrade go 1.19 and mention this there?

Reply:
I'd update to 1.19. don't see many reasons to not to

Reply:
I've created a JIRA ticket, but haven't filled in any details: https://polygon.atlassian.net/browse/EVM-168

Reply:
I just commented and attached the link of this discussion so that we don't forgot to fix this when upgrading

Reply:
Closing this discussion.